doc/api/process.md

```diff
@@ -1730,6 +1730,41 @@ function definitelyAsync(arg, cb) {
 }
 ```
 
+### When to use `queueMicrotask()` vs. `process.nextTick()`
+
+The [`queueMicrotask()`][] API is an alternative to `process.nextTick()` that
+also defers execution of a function using the same microtask queue used to
+execute the then, catch, and finally handlers of resolved promises. Within
+Node.js, every time the "next tick queue" is drained, the microtask queue
+is drained immediately after. The timing of queued microtasks is nearly
```

Comment:
> The timing of queued microtasks is nearly
identical to `process.nextTick()` with the notable exception that the next
tick queue always executes first.

Doesn't the sentence before this already say the same thing?

Reply:
Probably lol. I'll take another edit pass ;)